app/components/candidate_interface/application_form_course_choices_component.html.erb

```diff
@@ -1,19 +1,21 @@
-<h2 class="govuk-heading-m govuk-!-font-size-27 govuk-!-margin-bottom-2">
-  <%= page_title %>
-</h2>
+<section class="govuk-!-margin-bottom-8">
+  <h2 class="govuk-heading-m govuk-!-font-size-27">
+    <%= page_title %>
+  </h2>
```

Comment:
Given how we’re indenting headings elsewhere on this page, and that this is a short line of code:

```suggestion
  <h2 class="govuk-heading-m govuk-!-font-size-27"><%= page_title %></h2>
```

Reply:
👍